auslib/db.py

```diff
@@ -1943,7 +1955,9 @@ def update(self, where, what, changed_by, old_data_version, transaction=None, dr
         current_releases = self.select(where=where, columns=[self.name, self.product, self.read_only], transaction=transaction)
         for current_release in current_releases:
             name = current_release["name"]
-            if "product" in what or "data" in what:
+            is_readonly_change = "read_only" in what and current_release["read_only"] != what["read_only"]
+
+            if not is_readonly_change and ("product" in what or "data" in what):
```

Comment:
I realize there's no way easy way to do it at this time, but I think there's a potential bug here. If both read only _and_ data are changing, we'll bypass the `_proceedIfNotReadOnly` check.

Reply:
Yeah, this is a problem once `data` can't be null in SC table, so I assume that readonly will always be changed independently. 

Do you think that it is possible to identify only the readonly change using other approach ?

I don't remember well (because I was away for a few weeks), probably it is the one of reasons I tried manage readonly in other table than release table, in the other PR.

Reply:
Can we just leave the conditional statement as `if not is_readonly_change` or is there some reason we can't run `_proceedIfNotReadOnly` if `product` or `data` is in `what`? That way, if for some reason `product` or `data` change alongside it in the future, we won't accidentally skip the `_proceedIfNotReadOnly` check.